src/get-started/wordcount-example.md

```diff
@@ -769,14 +770,14 @@ PCollection<KV<String, Long>> wordCounts = windowedWords.apply(new WordCount.Cou
 ```
 
 ```py
-This feature is not yet available in the Beam SDK for Python.
+# This feature is not yet available in the Beam SDK for Python.
 ```
 
 ### Write Results to an Unbounded Sink
 
 When our input is unbounded, the same is true of our output `PCollection`. We need to make sure that we choose an appropriate, unbounded sink. Some output sinks support only bounded output, while others support both bounded and unbounded outputs. By using a `FilenamePolicy`, we can use `TextIO` to files that are partitioned by windows. We use a composite `PTransform` that uses such a policy internally to write a single sharded file per window.
 
-In this example, we stream the results to a BigQuery table. The results are then formatted for a BigQuery table, and then written to BigQuery using BigQueryIO.Write.
+In this example, we stream the results to a BigQuery table. The results are then formatted for a Google BigQuery table, and then written to BigQuery using `BigQueryIO.Write`.
```

Comment:
The causal relationships here seem broken.

Reply:
Updated